src/pages/QuizPage.tsx

```diff
@@ -0,0 +1,210 @@
+import { useState, useEffect } from "react";
+import "./QuizPage.scss";
+import GameOver from "../components/quiz/GameOver";
+import QuizPassed from "../components/quiz/QuizPassed";
+import QuizStart from "../components/quiz/QuizStart";
+import QuizQuestion from "../components/quiz/QuizQuestion";
+import { Question, getRandomQuestions } from "../data/quizData";
+
+import { useAnimate } from "framer-motion";
+
+const gameSettings = {
+  numOfQuestions: 10,
+  maxTime: 30,
+  maxVw: 72,
+  startVw: 10,
+  wrongTime: 3,
+  correctTime: 3,
+};
+
+function QuizPage() {
+  const [seconds, setSeconds] = useState(gameSettings.maxTime);
+  const [currentQuestion, setCurrentQuestion] = useState(0);
+  const [score, setScore] = useState(0);
+  const [firstIncorrect, setFirstIncorrect] = useState(true);
+  const [gameState, setGameState] = useState<
+    "gameMenu" | "gameRunning" | "gameOver" | "gameWon"
+  >("gameMenu");
```

Comment:
Split this into a type defined at the top of the file, like
```ts
type GameState = "gameMenu" | "gameRunning" | "gameOver" | "gameWon";
```
then this can become
```suggestion
  const [gameState, setGameState] = useState<GameState>("gameMenu");
```